src/basic/efivars.c

```diff
@@ -96,6 +96,17 @@ int efi_get_variable(
                                 (void) usleep_safe(EFI_RETRY_DELAY);
                 }
 
+                /* According to comment of linux source code, efivarfs represents uncommitted
+                 * variables with zero-length files. Reading them should return EOF, even though
+                 * st.st_size > 0
+                 *
+                 * See https://github.com/systemd/systemd/blob/b6b8527cd184085008585c1a1f9725eb97c342ef/src/basic/efivars.c#L99-L101
```

Comment:
No need to source out to GitHub for our own code. Just drop it.

Reply:
I give a wrong link. I wanted to include a link to the linux source code, I'll change the link.